### PR TITLE
cloudstack: new module cs_disk_offering

### PR DESCRIPTION
**Pull request by dpassante**
##### SUMMARY

New module cs_disk_offering.

Mainly a copy/paste from @resmo 's "cs_service_offering" module.

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py

##### ANSIBLE VERSION

```
2.7
```

### CODE REVIEW

File: lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+---
+module: cs_disk_offering
+description:
+  - Create and delete disk offerings for guest VMs.
+  - Update display_text or display_offering of existing disk offering.
+short_description: Manages disk offerings on Apache CloudStack based clouds.
+version_added: "2.7"
+author:
+  - "David Passante (@dpassante)"
+  - "René Moser @resmo)"
```
Comment (resmo) [inline lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py]: missing (

File: lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: cs_disk_offering
+description:
+  - Create and delete disk offerings for guest VMs.
```
Comment (resmo) [inline lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py:19]: , update

**Review by resmo — CHANGES_REQUESTED** [review]
minor things otherwise good job

File: lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py (hunk 2 above)
Comment (dpassante) [reply lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py:19]: I put the update description in the following line but in fact it wasn't really a good idea :)

File: lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py (hunk 1 above)
Comment (dpassante) [reply lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py]: oops!

File: lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py (hunk 2 above)
Comment (resmo) [reply lib/ansible/modules/cloud/cloudstack/cs_disk_offering.py:19]: ah, well. that works for me

**Review by resmo — APPROVED** [review]
shipit

**Pull request merged**